### DISCUSSION TITLE
How do you set focus on a form field?

### DISCUSSION THREAD

**Discussion by seabasss** [danharrin/alpine-mousetrap#2] [General]
How do I set focus on a form field? Just a `x-mousetrap.ctrl+f` on the input doesn't seem to activate it.

Something like this, but guess I need a different way to access it other than `this`:
```
<input x-mousetrap.ctrl+f="this.focus()" />
```
Thanks!

**Comment by seabasss** [comment]
This seems to do the trick:
```
<input x-ref="input" x-mousetrap.ctrl+f="$refs.input.focus()">
```

Let me know if there's a better way, or if I can use the magic id attribute for this somehow.

**Reply by danharrin** [reply]
`$el` instead of `$refs.input`

**Reply by seabasss** [reply]
Awesome! Sorry for the trivial question.

**Reply by danharrin** [reply]
No worries!
